timed/reports/management/commands/notify_reviewers_unverified.py

```diff
@@ -80,13 +96,17 @@ def _get_unverified_reports(self, start, end):
 
         return queryset
 
-    def _notify_reviewers(self, start, end, reports):
+    def _notify_reviewers(self, start, end, reports, message, cc):
         """Notify reviewers on their unverified reports."""
         User = get_user_model()
         reviewers = User.objects.all_reviewers().filter(email__isnull=False)
         subject = '[Timed] Verification of reports'
         from_email = settings.DEFAULT_FROM_EMAIL
-        mails = []
+        connection = get_connection()
+        messages = []
+        match = re.match('[^@]+@[^@]+\.[^@]+', ', '.join(cc))
```

Comment:
Is this really necessary? If we're "validating" mail addresses, we should also tell the user if it failed..
I suggest leaving this out completely.